scripts/httpd/upload.cgi

```diff
@@ -122,11 +122,54 @@ def ensure_namespaced(name, namespace):
     return name
 
 
-def main():
-    form = cgi.FieldStorage()
+def get_checksum_and_hash_type(form):
+    # Search for the file hash, start with stronger hash functions
+    if 'sha512sum' in form:
+        checksum = check_form(form, 'sha512sum')
+        hash_type = "sha512"
+
+    elif 'md5sum' in form:
+        # Fallback on md5, as it's what we currently use
```

Comment:
Can you, please, fix this comment to only say "Fallback on md5".

Reply:
Sure, updated.